crates/turbopack-ecmascript-runtime/js/src/dev/runtime/dom/runtime-backend-dom.ts

```diff
@@ -52,6 +52,11 @@ async function loadWebAssemblyModule(
   return await WebAssembly.compileStreaming(req);
 }
 
+// [TODO] need to match behavior as similar to UrlAssetReference
```

Comment:
Don't we need to implement this now?

Reply:
I'd like to separate PR, since this is dom runtime implementation will likely entirely different - it shouldn't return file:// path anyway.